README.md

```diff
@@ -102,6 +106,23 @@ There are two key techniques we will be using to wrap the Job Library through th
 2. We'll use the **CommonName** that has been verified through our CA chain and the TLS handshake to authenticate user. We will index our `authZService` using the CommonName stored in the client cert. This can be found here: `r.TLS.VerifiedChains[0][0].Subject.CommonName`.
 We will make the assumption that the CommonName will always be unique to the client and that CAs will only sign the correct clients.
 
+## Security - Transport
+This implementation will be using TLS 1.3 as it's the most recent implementation.
```

Comment:
Could you elaborate a little on why you chose v1.3? What are some of the benefits over the previous v1.2?

Reply:
I don't know too much about the benefits, I chose it because it's new.

Reply:
I added some stuff regarding speed-ups that I found online. I wouldn't really be able answer in any more intelligent way without more research. I didn't even realize TLS 1.3 was out, thought 1.2 was the current one.